docs/api-docs/building-storefronts/abandoned-carts.md

```diff
@@ -0,0 +1,289 @@
+
+![Complete your purchase email](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Abandoned%20Carts/01-abandoned-carts.png "Complete your purchase email")
+
+To enable the Abandoned Cart Saver notifications on your store, go to **Store Setup > Store Settings > Miscellaneous** in the control panel. To access your Abandoned Cart Emails settings, visit **Marketing > Abandoned Cart Emails**.
+
+![Abandoned cart emails](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Abandoned%20Carts/02-abandoned-carts.png "Abandoned cart emails")
+
+### Triggering Abandoned Cart Saver notifications
+
+The Abandoned Cart Saver will trigger under the following circumstances:
+- **Stencil stores**: a logged-in customer adds an item to the cart, then leaves the store without attempting payment.
+- **Stores using [Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout?language=en_US)**: a guest customer adds an item to the cart, begins checkout by entering their email address, then leaves the store without attempting payment.
+- **Headless stores using custom checkout solutions**: a logged-in customer with the `accepts_product_review_abandoned_cart_email` property set to `true` adds an item to the cart, then leaves the store without attempting payment.
+
+## Setting up the abandoned cart route
+
+The Abandoned Cart Saver email link points to the Stencil storefront (channel ID 1) by default. To point the Abandoned Cart Saver email link to your headless storefront, you need to set up a `recover_abandoned_cart` site route.
+
+This operation requires a channel site ID. If you do not know your channel site ID, you can retrieve it by sending a `GET` request to the [Get a Channel Site](https://developer.bigcommerce.com/api-reference/store-management/channels/channel-site/get-channel-site) endpoint.
+
+```http
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels/{channel_id}/site
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/channels/channel-site/get-channel-site#requestrunner)
+
+**Example response**
+
+```json
+{
+  "data": {
+    "channel_id": 773240,
+    "created_at": "2021-08-23T20:02:11Z",
+    "id": 1001,
+    "updated_at": "2021-08-23T20:02:11Z",
+    "url": "http://commerce-zr8y-teststore-bigcommerce.vercel.app"
+  },
+  "meta": {}
+}
+```
+
+You can locate your channel ID by:
```

Comment:
There is something that it doesn't like about the page breaks so GitHub isn't rendering the MD correctly. The following seemed to work, but you might need to play around with it: 

```md
You can locate your channel ID by:

- going to **Channel Manager > Storefronts** in the control panel, clicking on **...** next to your headless storefront, and selecting **Advanced settings** from the dropdown

  **or**

- sending a `GET` request to the [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels) endpoint. 
```
You can locate your channel ID by:

- going to **Channel Manager > Storefronts** in the control panel, clicking on **...** next to your headless storefront, and selecting **Advanced settings** from the dropdown

  **or**

- sending a `GET` request to the [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels) endpoint.

Reply:
I removed the page breaks completely.